src/containers/MyNdla/components/UserInfo.tsx

```diff
@@ -7,60 +7,80 @@
  */
 
 import { useTranslation } from "react-i18next";
-import styled from "@emotion/styled";
-import { misc, spacing } from "@ndla/core";
-import { Text } from "@ndla/typography";
+import { Text } from "@ndla/primitives";
+import { styled } from "@ndla/styled-system/jsx";
 import { GQLMyNdlaPersonalDataFragmentFragment } from "../../../graphqlTypes";
 
 interface Props {
   user: GQLMyNdlaPersonalDataFragmentFragment | undefined;
 }
 
-const StyledComponentContainer = styled.div`
-  display: flex;
-  flex-direction: column;
-  gap: ${spacing.normal};
-  max-width: ${misc.maxTextWidth};
-`;
+const StyledComponentContainer = styled("div", {
+  base: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "medium",
+  },
+});
 
-const ShortInfoDiv = styled.div`
-  display: flex;
-  flex-direction: column;
-  gap: ${spacing.xxsmall};
-`;
+const ShortInfoDiv = styled("div", {
+  base: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "4xsmall",
+    maxWidth: "surface.xlarge",
+  },
+});
 
-const StyledUl = styled.ul`
-  padding-left: ${spacing.mediumlarge};
-  padding-bottom: ${spacing.small};
-`;
+const StyledUl = styled("ul", {
+  base: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "xsmall",
+    listStyleType: "unset",
+    paddingBlockEnd: "xsmall",
+    paddingInlineStart: "xlarge",
+  },
+});
+
+const UserInfoWrapper = styled("span", {
+  base: {
+    display: "flex",
+    flexDirection: "row",
+    gap: "4xsmall",
+  },
+});
+
+const ShortInfoLine = ({ label, value }: { label: string; value?: string }) => (
+  <UserInfoWrapper>
+    <Text textStyle="body.large">
+      <strong>{label}:</strong>
+    </Text>
+    <Text textStyle="body.large">{value}</Text>
```

Comment:
Dette trenger vel ikke å være to forskjellige `Text`-elementer?

Reply:
Må sikkert ikke, men gjør vel heller ingenting om det er, ettersom dette er tittel-verdi sammenheng. Er også en bonus i det at en skjermleser (har verifisert med NVDA) vil kunne lese tittel og verdi separat fra hverandre (husker vi hadde et issue på å få splittet opp noe for en tid sida, hvor løsningen ble å legge dem som separate elementer).

Reply:
I dette tilfellet ønsker vi vel ikke at det skal leses opp som to separate ting?